test/src/test/java/org/neo4j/ogm/autoindex/BaseAutoIndexManagerTestClass.java

```diff
@@ -102,9 +107,15 @@ public void setUp() throws Exception {
     }
 
     @After
-    public void tearDown() throws Exception {
-        executeDrop(definition);
-        executeDrop(statements);
+    public void dropIndexesAndConstraints() throws Exception {
+        org.neo4j.ogm.model.Result result = session.query("call db.constraints()", emptyMap());
+        result.queryResults().forEach( it -> {
+            session.query("DROP " + it.get("description"), emptyMap());
```

Comment:
This will fail if there is a character in label or property of the index/constraint (the returned description is not escaped). What was wrong with the previous code?

Reply:
actually I had strange behaviors (very slow tests, locks, errors) but maybe it was due to the facts index requests were not done in separate transactions.
I'll try to revert to the old version see if it works.